vm/devices/storage/disk_layered/src/lib.rs

```diff
@@ -519,7 +513,10 @@ pub trait LayerIo: 'static + Send + Sync + Inspect {
     /// committed to disk.
     fn is_fua_respected(&self) -> bool;
 
-    /// Returns true if the layer is read only.
+    /// Returns true if the layer is logically read only.
+    ///
+    /// If this returns true, the layer might still be writable via
+    /// `write_no_overwrite`, used to populate the layer as a read cache.
     fn is_read_only(&self) -> bool;
```

Comment:
kinda feels like its worth renaming the method to `is_logically_read_only` as well?

Reply:
Sure